python/cudf/cudf/pandas/_wrappers/numpy.py

```diff
@@ -162,6 +163,8 @@ def ndarray__array_ufunc__(self, ufunc, method, *inputs, **kwargs):
     slow_to_fast=lambda slow: cupy.asarray(slow).flat,
     additional_attributes={
         "__array__": array_method,
+        "__reduce__": _FinalProxy.__reduce__,
```

Comment:
Hmm we wouldn't need to add this as an additional attribute to `ndarray` above?

Reply:
Yes, thanks for catching!